src/wp-admin/security.php

```diff
@@ -0,0 +1,157 @@
+  <h1><?php echo esc_html( $title ); ?></h1>
+  <h2 class="nav-tab-wrapper">
+<?php   foreach ($tabs as $tab => $info): ?>
+    <a href="/wp-admin/security.php?tab=<?=$tab?>" class="<?=join(' ', $info['classes'])?>"><?=$info['title']?></a>
+<?php   endforeach; ?>
+  </h2>
+<?php
+switch ($active_tab):
+    case 'users':
+?>
+
+  <div class="card">
+    <h2><?php _e( '"Security First"' ); ?></h2>
+    <p><?php _e( "Security is important to business and it’s important to us here at ClassicPress. By bringing security forward to a place of greater prominence within the admin interface, we create a more streamlined experience for both users and developers." ); ?></p>
+    <p><?php _e( "As ClassicPress continues to evolve, the Security page will become the hub for all security features for ClassicPress core and 3<sup>rd</sup> party plugins that choose to support it." ); ?></p>
+    <p><?php printf(
+        /* translators: canonical link to security forum */
+        __( "Watch this page and the ClassicPress <a href=\"%s\" target=\"_blank\">Security forum</a> for more news as development continues." ),
+        'https://forums.classicpress.net/security'
+    ); ?></p>
+  </div>
+
+<?php
+        break;
+    case 'developers':
+?>
+
+  <div id="poststuff">
+    <div id="post-body" class="metabox-holder columns-2">
+      <div id="post-body-content">
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'Overview' ); ?></span></h2>
+            <div class="inside">
+              <div>
+                <p>The Security page is the first visible step in improving the overall approach to security in ClassicPress. Its pupose is to solve two related problems: <strong>discovery</strong> and <strong>auditing</strong>.</p>
+                <p>Most plugins are organised around their core purpose; after all, that purpose is the reason they were installed. Unfortunately this leads to poor discoverability of security-related settings - they may be in there somewhere, there may be none at all - without looking through everything there&lsquo;s no way to know.</p>
+                <p>Currenty, auditing the overall security profile of a ClassicPress site is impractical if there are more than a few plugins. Having all security-related settings from all plugins in one place means those settings can be autidted far more easily, as the time taken will be proportional to the number of settings, not the number of places to look for those settings.</p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'API' ); ?></span></h2>
+            <div class="inside">
+              <p>Notes: I left $capability for consistency; we should probably force it to 'manage_options' or just drop it completely - discuss.</p>
+              <!-- HTML generated using hilite.me -->
```

Comment:
Same as https://github.com/ClassicPress/ClassicPress/pull/475#discussion_r321409132, this needs to be documented in comments so that someone looking to modify this section can re-create the work.

>Which color scheme was used on hilite.me? Were there any modifications made to the resulting inline CSS?
>default, no.

From past discussion the opening `<?php` tag needed to get hilite.me to colorize PHP code was removed. Let's document that, the name of the default color scheme, and any other steps needed to reproduce this example using hilite.me, with the goal of making this easier for the next person who looks at it.

Reply:
I've added a comment at the top of the code block - hopefully that's enough.

Reply:
LGTM, thanks!